pkg/cmd/roachtest/cluster.go

```diff
@@ -209,7 +209,17 @@ func unregisterCluster(c *cluster) bool {
 	return exists
 }
 
-func execCmd(ctx context.Context, l *logger, args ...string) error {
+// execCmd runs a command. stdout/stderr can be passed in to capture the
```

Comment:
It's annoying that these new arguments had to be plumbed through so many places. I think it would be better to rename this to execCmdWithOutput and keep a backwards-compatible shim for execCmd.